### ISSUE TITLE
[Warrior] Prot war , 10% dmg talent not working

### ISSUE THREAD

**Issue by Manpower99** [GameFreedomGG/Sindragosa#523]
Decription:
Talent by the name of One-Handed Weapon Specialization is not working at all 

How it works:
Does not work at all , regardless of how many talent points you put into it, loss of damage/threat 
if u mouseover melee damage on your character tab , says 95%

How it should work:
While a one hander is equipped , in protection stance a warrior should do 10% more damage ( and thus threat increase )
whenever you mouseover melee damage in character tab , should be 105%

Source (you should point out proofs of your report, please give us some source)
(https://user-images.githubusercontent.com/100322493/155432604-a08f1f91-3f71-486c-b2fd-76ad2c17c0ff.jpg)

**Comment by UltraNix** [comment]
This "x95%" is not this talent but other dmg reduction aura you have - Defensive Stance.

**Comment by UltraNix** [comment]
No. You won't see that in char sheet as colored number. Just respec or remove that talent and you will see the change in your char sheet
